helm/inteli-api/templates/deployment.yaml

```diff
@@ -23,6 +23,11 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
           command: [ 'npx', 'knex', "migrate:latest", "--env", "production" ]
           env:
+            - name: POLKADOT_API
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ include "inteli-api.fullname" . }}-config
+                  key: polkadotApi
```

Comment:
Should this be here?  Its an init container.

Reply:
it should, thanks. Marking as resolved as this is no longer relevant